core/src/main/resources/defaults/blueprints/7.2.9/cdp-opdb.bp

```diff
@@ -272,6 +272,28 @@
         ],
         "displayName": null,
         "roles": []
+      },
+      {
+        "refName": "queuemanager",
```

Comment:
did we tested this ?

Reply:
Yes, as stated in the description, this was tested by deploying on a mow-int environment.